beacon_node/lighthouse_network/src/peer_manager/peerdb/client.rs

```diff
@@ -193,6 +185,14 @@ fn client_from_agent_version(agent_version: &str) -> (ClientKind, String, String
             }
             (kind, version, os_version)
         }
+        Some("erigon") => {
```

Comment:
Should also include `caplin` for in case they change to the actual client's name?

Reply:
I looked on mainnet at all the unknown identify messages. Caplin's agent-string is: "erigon/caplin", which is what I've hard-coded here.